### DISCUSSION TITLE
[nitpick] Discrepancy between #clients shown in tab bar bubble vs in the text

### DISCUSSION THREAD

**Discussion by sagz** [Mygod/VPNHotspot#328] [Q&A]
![Screenshot_20220130-212522](https://user-images.githubusercontent.com/964843/151749545-53322285-bd29-4959-9c82-d999bff73e4c.png)
What causes this issue? :)

**Comment by Mygod** [comment]
The bottom counter counts all clients across all tethering interfaces. Also they use different mechanisms but you should consider the top counter to be more accurate.
